arguments_test.go

```diff
@@ -72,6 +75,15 @@ func TestJSONInputOutput(t *testing.T) {
 	assert.Equal(t, 20756, outObj.Value)
 }
 
+func TestReadNotEmpty(t *testing.T) {
+	// Note: The contents need to be larger than the default buffer size of bufio.NewReader.
+	r := bytes.NewReader([]byte("{}" + strings.Repeat("{}\n", 10000)))
```

Comment:
so the "{}" is the 'end' of the thrift request right?  Which is why anything after the first '{}' causes an error?